commit 568f587734a6f24f55717a76b06f53bd55696d70 (refs/changes/04/904/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-11-15T17:26:24+00:00 (2 years, 11 months ago)
    
    OS-5793 illumos build failures point to wrong files
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/tools/build_illumos b/tools/build_illumos
index b84389d1..c362cb80 100755
--- a/tools/build_illumos
+++ b/tools/build_illumos
@@ -65,8 +65,8 @@ time ./nightly illumos.sh
 status=$?
 if [[ $status -ne 0 ]]; then
 	log_dir=$(ls -1t ./log | head -n1)
-	mail_msg=$PWD/$log_dir/mail_msg
-	nightly=$PWD/$log_dir/nightl.log
+	mail_msg=$PWD/log/$log_dir/mail_msg
+	nightly=$PWD/log/$log_dir/nightly.log
 
 	#
 	# Note, the blank lines at the start and end of the heredoc are
